### ISSUE TITLE
Show Half Event Width on Start date and End date

### ISSUE THREAD

**Issue by maaqib121** [fullcalendar/fullcalendar#5933]
### Checklist

Please mark these items with an [x]

  - [x] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [x] Other people will find this feature useful

### Feature Description

I am working on a hotel management system and want to show bookings against every room on full calendar monthly view.

Now suppose, we have a booking from November 1, 2020 to November 7, 2020. In this case, the event start date will be November 1, 2020 and end date will be November 7, 2020. As full calendar shows the event inclusive of start date and exclusive of end date, the event will be shown from November 1, 2020 to November 6, 2020.

What I want to show (in monthly view and resource timeline monthly view) is:
1. Half width of event on start date (to highlight it as check in date)
2. Half width of event on end date (to highlight it as check out date)

Is there a way to achieve this currently?

**Comment by acerix** [comment]
You could do that with CSS, something like this:

https://codepen.io/acerix/pen/xxOjoyg?editors=0110

**Comment by maaqib121** [comment]
Thanks. It's working very nicely. Although, when we decrease screen size, the calendar slots shrink and the results become invalid. I tried to solve this issue by using % but it starts to show invalid results as well when two events are overlapping.

The first image attached is the result what I want on all screens.

The second image shows when the screen size drops to less than 690px, the results become invalid.

In third image, I have changed px to % and if we look on Pine Park and Pearl Continental event, both are ending on the same dates but not finishing at the same place

![Original](https://user-images.githubusercontent.com/55296753/98245188-5f54a080-1f92-11eb-84ad-4ef900055b8f.png)
![With 65px margins](https://user-images.githubusercontent.com/55296753/98245191-61b6fa80-1f92-11eb-9e74-9324c17e8872.png)
![With Percentage](https://user-images.githubusercontent.com/55296753/98245194-62e82780-1f92-11eb-933d-1245ea59fddc.png)
